packages/@okta/vuepress-site/docs/guides/refresh-tokens/refresh-token-rotation/index.md

```diff
@@ -0,0 +1,62 @@
+
+* `app.oauth2.as.token.detect_reuse` for [Custom Authorization Servers](/docs/concepts/auth-servers/#custom-authorization-server)
+* `app.oauth2.token.detect_reuse` for the [Org Authorization Server](/docs/concepts/auth-servers/#org-authorization-server)
+
+## Grace period for token rotation
+
+Token reuse detection can sometimes impact the user experience. For example, when apps are accessed by users with a poor network connection, new tokens issued by Okta might not reach the client app. As a result, the client might want to reuse the refresh token to get new tokens from Okta. So, Okta offers a grace period when you [configure refresh token rotation](#enable-refresh-token-rotation). After the refresh token is rotated, the previous token remains valid for the configured amount of time to allow clients to get the new token.
+
+## Enable refresh token rotation
+
+Rotating refresh token behavior is the default for SPAs. When you create a new SPA, or when you update an existing SPA, and select **Refresh Token** as the allowed grant type, rotating the refresh token is set as the default.
+
+To update existing OpenID Connect applications to use refresh token rotation:
+
+1. From the Developer Console, click **Applications** and then select the application that you want to configure.
+2. Select the **General** tab and click **Edit**.
+3. In the **Allowed grant type** section, select **Refresh Token**.
+4. In the **REFRESH TOKEN** section that appears, select **Rotate token after every use**.
+5. Make any adjustments to the number of seconds for the **Grace period for token rotation**. You can change the value to any number between 0 and 60 seconds. After the refresh token is rotated, the previous token remains valid for this amount of time to allow clients to get the new token.
+
+When you create a new native or web app and want to use refresh token rotation:
+
+1. From the Developer Console, click **Applications** and then **Add Application**.
+2. While walking through the app creation wizard, select **Refresh Token** in the **Allowed grant type** section.
+3. After you click **Done**, select the **General** tab and click **Edit**.
+4. In the **REFRESH TOKEN** section, select **Rotate token after every use**.
+5. Make any adjustments to the number of seconds for the **Grace period for token rotation**. The default is set to 30 seconds. You can change the value to any number between 0 and 60 seconds. After the refresh token is rotated, the previous token remains valid for this amount of time to allow clients to get the new token.
+
+### Refresh token rotation properties
+
+After you enable refresh token rotation, the `refresh_token` property appears within `settings.oauthClient` for your app. The `refresh_token` property is an array that contains the `rotation_type` and `leeway` parameters. Accepted values for `rotation_type` are `ROTATE` or `STATIC`. The accepted value for `leeway` is any number between 0 and 60.
```

Comment:
`The `refresh_token` property is an array that contains the `rotation_type` and `leeway` parameters.` -> `The `refresh_token` property is an object that contains the `rotation_type` and `leeway` properties.`